bazel/src/test/data/googleapis/google/example/library/type/BUILD.bazel.baseline

```diff
@@ -35,7 +35,7 @@ java_proto_library(
 
 # Open Source Packages
 java_gapic_assembly_gradle_pkg(
-    name = "library-types-java",
+    name = "google-example-library-types-java",
```

Comment:
We may never have the use case but I guess it would not work if we have two versioned types-only packages?

Reply:
If they have different versions, then they will have different proto packages and thus different target names - this just happens to be an unversioned package in the test.

`google.library.types.v1` --> `google-library-types-v1-java`
`google.library.types.v2` --> `google-library-types-v2-java`

Or am I misunderstanding the case?

Reply:
Yes, you are right, I missed that the proto package would include versions and this test just happen to not have versions.